pkgs/by-name/br/bruno/package.nix

```diff
@@ -1,61 +1,127 @@
 { lib
-, stdenv
-, fetchurl
-, autoPatchelfHook
-, dpkg
-, wrapGAppsHook
-, alsa-lib
-, gtk3
-, mesa
-, nspr
-, nss
-, systemd
+
+, fetchFromGitHub
+, buildNpmPackage
 , nix-update-script
+, electron-bin
+, writeShellScriptBin
+, makeWrapper
+, copyDesktopItems
+, makeDesktopItem
+, pkg-config
+, pixman
+, cairo
+, pango
+, npm-lockfile-fix
 }:
 
-stdenv.mkDerivation rec {
+let
+  electron = electron-bin;
+in
```

Comment:
Whats the reason behind this?

Reply:
Only convenience. I was testing with different electron versions at first and settled for the default one.